Source/SG/SG.cs

```diff
@@ -98,22 +133,66 @@ public static PAENConfiguration GetDefault()
         public static void ConfigureInstrument(NIRfsg rfsgHandle, InstrumentConfiguration instrConfig)
         {
             rfsgHandle.SignalPath.SelectedPorts = instrConfig.SelectedPorts;
-            rfsgHandle.RF.ExternalGain = -instrConfig.ExternalAttenuation_dBm;
+            rfsgHandle.FrequencyReference.Source = RfsgFrequencyReferenceSource.FromString(instrConfig.ReferenceClockSource);
             rfsgHandle.RF.Configure(instrConfig.CarrierFrequency_Hz, instrConfig.DutAverageInputPower_dBm);
+            rfsgHandle.RF.ExternalGain = -instrConfig.ExternalAttenuation_dBm;
 
-            rfsgHandle.FrequencyReference.Source = RfsgFrequencyReferenceSource.FromString(instrConfig.ReferenceClockSource);
-            
-            if (instrConfig.ShareLOSGToSA)
+            string instrumentModel = rfsgHandle.Identity.InstrumentModel;
+
+            if (Regex.IsMatch(instrumentModel, "NI PXIe-5(82.|645R)")) // matches on any baseband instrument without an LO
+                return; // return early since the instrument doesn't have any LOs that can be configured

```

Comment:
same as above, do we really support 5646R as basedeband?

Reply:
excellent point. i'd say, no.

Reply:
Same comment as above; let's only make this section for supported VSTs; return for everything else.

Reply:
removed this check